client/src/components/Details.js

```diff
@@ -15,9 +15,10 @@ class Details extends React.Component {
   state = { cardDetailsData: null };
 
   componentDidMount() {
-    const { nodeData } = this.props;
+    const { records } = this.props;
     const queryVals = queryString.parse(this.props.location.search);
```

Comment:
Here is an example of React Router's location prop fyi.

Reply:
See above comment